### PR TITLE
Make the redpanda examples work with deephaven v2 on Mac

### LINKED ISSUE #2181: Migrate manual redpanda tests scripts to new deephaven python wrappers.
The python code under `redpanda` and `redpanda-apicurio` has a big comment preamble describing tests that can be run.  Those tests need to be converted to use the new python wrappers.

### PR DESCRIPTION
**Pull request by jmao-denver**
Fixes #2181

### CODE REVIEW

File: redpanda/examples/post-share-price-schema.sh [hunk 1]
```diff
@@ -5,7 +5,9 @@
 # redpanda schema service does not support newlines, and needs the schema embedded in a json object as a
 # value for a "schema" key, which means the keys and values in the schema itself need to have their
 # quotes escaped... what a royal pain.
-PAYLOAD=$(echo -n '{ "schema" : "'; cat avro/share_price.json | sed 's/"/\\"/g' | tr '\n' ' '; echo -n '" }')
+PAYLOAD=$(echo '{ "schema" : "\c'; cat avro/share_price.json | sed 's/"/\\"/g' | tr '\n' ' '; echo '" }\c')
```
Comment (jcferretti) [inline redpanda/examples/post-share-price-schema.sh:8]: It doesn't work to change something to work on mac if it ends up broken in linux...

```
cfs@erke 20:58:58 ~/dh/oss1/deephaven-core/redpanda/examples
$ PAYLOAD=$(echo '{ "schema" : "\c'; cat avro/share_price.json | sed 's/"/\\"/g' | tr '\n' ' '; echo '" }\c')

cfs@erke 20:59:07 ~/dh/oss1/deephaven-core/redpanda/examples
$ echo $PAYLOAD 
{ "schema" : "\c { \"type\" : \"record\", \"namespace\" : \"io.deephaven.examples\", \"name\" : \"share_price\", \"fields\" : [ { \"name\" : \"Symbol\", \"type\" : \"string\" }, { \"name\" : \"Side\", \"type\" : \"string\" }, { \"name\" : \"Qty\", \"type\" : \"int\" }, { \"name\" : \"Price\", \"type\" : \"double\" } ] } " }\c
```

Sounds like in linux it needs `-e`, do you have `-e` on mac? (I assume you don't have -n this the change?)
```
cfs@erke 21:00:16 ~/dh/oss1/deephaven-core/redpanda/examples
$ PAYLOAD=$(echo -e '{ "schema" : "\c'; cat avro/share_price.json | sed 's/"/\\"/g' | tr '\n' ' '; echo -e '" }\c')

cfs@erke 21:00:25 ~/dh/oss1/deephaven-core/redpanda/examples
$ echo $PAYLOAD 
{ "schema" : "{ \"type\" : \"record\", \"namespace\" : \"io.deephaven.examples\", \"name\" : \"share_price\", \"fields\" : [ { \"name\" : \"Symbol\", \"type\" : \"string\" }, { \"name\" : \"Side\", \"type\" : \"string\" }, { \"name\" : \"Qty\", \"type\" : \"int\" }, { \"name\" : \"Price\", \"type\" : \"double\" } ] } " }
```

File: redpanda/examples/python/kafka-produce-avro.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 # $ cd confluent-kafka
 # $ source bin/activate
 # $ pip3 install confluent-kafka
+# $ pip3 install avro
```
Comment (jcferretti) [inline redpanda/examples/python/kafka-produce-avro.py:13]: Same comment as above, would it make sense to make this part of the note for mac below?

File: redpanda/examples/python/kafka-produce-avro.py [hunk 3]
```diff
@@ -23,25 +24,26 @@
...
+#    > from deephaven import kafka_consumer as kc
+#    > from deephaven.stream.kafka.consumer import TableType
 #
 # == Example (1)
 #
 # Load a schema into schema registry for share_price_record.
 # From the command line in the host (not on a docker image), run:
 #
-#    $ sh ../post-share-price-schema.sh
+#    $ sh ./post-share-price-schema.sh
```
Comment (jcferretti) [inline redpanda/examples/python/kafka-produce-avro.py:35]: Same comment as above.

File: redpanda/examples/python/kafka-produce.py [hunk 4]
```diff
@@ -9,6 +9,7 @@
 # $ cd confluent-kafka
 # $ source bin/activate
 # $ pip3 install confluent-kafka
+# $ pip3 install avro
```
Comment (jcferretti) [inline redpanda/examples/python/kafka-produce.py:12]: Same as above.

**Review by jcferretti — DISMISSED** [review]

File: redpanda/examples/post-share-price-schema.sh (hunk 1 above)
Comment (jmao-denver) [reply redpanda/examples/post-share-price-schema.sh:8]: echo -n didn't work for me in MacOS, but the ending '\c' should work for both Linux and MacOS.

**Review by jcferretti — APPROVED** [review]

**Pull request merged**